src/components/CreateCourseRunPage/index.jsx

```diff
@@ -12,6 +12,9 @@ class CreateCourseRunPage extends React.Component {
   constructor(props) {
     super(props);
     this.handleCourseCreate = this.handleCourseCreate.bind(this);
+    // We do not want to accidentally persist an error from a different page
```

Comment:
Nit: If we want this comment to be tied to the method, we should place it there rather than copying it multiple times.